src/client/net/xhr.js

```diff
@@ -153,7 +153,8 @@ spf.net.xhr.send = function(method, url, data, opt_options) {
 
   // For POST, default to `Content-Type: application/x-www-form-urlencoded`
   // unless a custom header was given.
-  var addContentTypeFormUrlEncoded = (method == 'POST');
+  var isFormData = (data instanceof FormData);
```

Comment:
In browsers/versions that do not yet support `FormData`, this will throw something like:
```
Uncaught ReferenceError: FormData is not defined
```

Can you also add a check to make sure `FormData` is defined?  Something like the following should work nicely:

```js
'FormData' in window && data instanceof FormData
```